src/quo2/components/notifications/activity_logs.cljs

```diff
@@ -66,25 +106,24 @@
      body)])
 
 (defn- activity-buttons
-  [button-1 button-2]
-  (let [size         24
-        common-style {:padding-top    3
-                      :padding-right  8
-                      :padding-bottom 4
-                      :padding-left   8}]
+  [button-1 button-2 replying? reply-input]
+  (let [size         (if replying? 40 24)
+        common-style (when replying?
+                       {:padding-vertical 9
+                        :flex-grow        1
+                        :flex-basis       0})]
     [rn/view {:margin-top     12
-              :flex           1
               :flex-direction :row
               :align-items    :flex-start}
      (when button-1
        [button/button (-> button-1
                           (assoc :size size)
-                          (assoc-in [:style :margin-right] 8)
-                          (update :style merge common-style))
+                          (update :style merge common-style {:margin-right 8}))
```

Comment:
common-style will be nil if `replying?` was `false` or `nil`, Did we test that ?

Reply:
I did test exactly this scenario @ibrkhalil because I had the same question as you, but the UI looks alright. Clojure's `merge` function can take nil values, and they'll just be ignored/skipped. It's the whole _nil punning_ stuff. For example:

```clojure
(merge {:a 1} nil {:b 2})
; => {:a 1, :b 2}
```